src/Confluent.Kafka/Impl/SafeKafkaHandle.cs

```diff
@@ -236,6 +236,10 @@ internal int Flush(int millisecondsTimeout)
         internal int AddBrokers(string brokers)
         {
             ThrowIfHandleClosed();
+            if (brokers == null)
+            {
+                throw new ArgumentNullException("brokers", "Argument 'brokers' must not be null.");
```

Comment:
interesting, i didn't realize this constructor overload existed. now there's redundant info in the exception, so something here is probably not ideal, but i don't think it's a big deal, so this is fine.